protobuf-lite/src/test/java/io/grpc/protobuf/lite/ProtoLiteUtilsTest.java

```diff
@@ -130,7 +135,8 @@ public void testAvailable() throws Exception {
     assertEquals(proto.getSerializedSize(), is.available());
     is.read();
     assertEquals(proto.getSerializedSize() - 1, is.available());
-    while (is.read() != -1) {}
+    while (is.read() != -1) {
+    }
```

Comment:
Thanks! Both formats are perfectly acceptable. Generally we try to avoid changing code style in places unrelated to the PR, because "if you change it, you own it".

Reply:
I think all the formatting changes were actually previously okay as they were. I didn't bring it up to let things go smoothly. I had assumed there was a full-file formatter, which normally just causes a mess of things; it's better to only change the formatting of the things you are changing.

I think the `new byte[] {` to `new byte[]{` is actually wrong, as [there's a space "before any open curly brace"](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace) (with some exceptions that don't apply here).

Reply:
Probably my formatter is not configured properly... I'll revert the formatting changes in a sec